cmd/admin-handlers-users.go

```diff
@@ -399,14 +399,26 @@ func (a adminAPIHandlers) AddUser(w http.ResponseWriter, r *http.Request) {
 			AccountName:     parentUser,
 			Action:          iampolicy.CreateUserAdminAction,
 			ConditionValues: getConditionValues(r, "", parentUser, claims),
-			IsOwner:         owner,
+			IsOwner:         false,
 			Claims:          claims,
 		}) {
 			writeErrorResponseJSON(ctx, w, errorCodes.ToAPIErr(ErrAccessDenied), r.URL)
 			return
 		}
 	}
 
+	if implicitPerm && !globalIAMSys.IsAllowed(iampolicy.Args{
+		AccountName:     accessKey,
+		Action:          iampolicy.CreateUserAdminAction,
+		ConditionValues: getConditionValues(r, "", accessKey, claims),
+		IsOwner:         false,
+		Claims:          claims,
+		DenyOnly:        true, // check if changing password is explicitly denied.
```

Comment:
@harshavardhana I think it is better if we have a separate function called IsExplicitedDenied() as IsAllowed().. simpler to read than DenyOnly flag.

Reply:
Will add subsequently